src/emails/interfaces/get-email-options.interface.ts

```diff
@@ -25,3 +25,12 @@ export interface GetEmailResponse {
   data: GetEmailResponseSuccess | null;
   error: ErrorResponse | null;
 }
+
+export interface CancelScheduleResponse {
+  data: CancelScheduleResponseSuccess | null;
+  error: ErrorResponse | null;
+}
+
+export interface CancelScheduleResponseSuccess {
+  email_id: string;
+}
```

Comment:
We need to fix the returned here, it should match the `DELETE domain/:domain_id` for example

Reply:
Hm I'm not sure if this is the same. Deleting a domain is different than removing the schedule. If it was to have the same return, it would be something like this:
```
  id: string;
  object: 'schedule';
  deleted: boolean;
```

which looks weird, don't you think?

Reply:
That's correct.

Why does it look weird?

Reply:
because `schedule` isn't really an object, but ok I'll push the changes

Reply:
Got it, should it be `emails`?